lib/updateVersion.js

```diff
@@ -0,0 +1,355 @@
+'use strict';
+var Cesium = require('cesium');
+
+var getUniqueId = require('./getUniqueId');
+var findAccessorMinMax = require('./findAccessorMinMax');
+var loadGltfUris = require('./loadGltfUris');
+
+var Cartesian3 = Cesium.Cartesian3;
+var DeveloperError = Cesium.DeveloperError;
+var Quaternion = Cesium.Quaternion;
+var WebGLConstants = Cesium.WebGLConstants;
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+
+module.exports = updateVersion;
+
+var updateFunctions = {
+    '0.8' : glTF_0_8_to_glTF_1_0,
+    '1.0' : glTF_1_0_to_glTF_1_1
+};
+
+/**
+ * Update the glTF version to the latest (1.1), or targetVersion if specified.
+ * Applies changes made to the glTF spec between revisions so that the core library
+ * only has to handle the latest version.
+ *
+ * @param {Object} gltf A javascript object containing a glTF asset.
+ * @param {Object} [options] Options for updating the glTF.
+ * @param {String} [options.targetVersion] The glTF will be upgraded until it hits the specified version.
+ * @returns {Object} The updated glTF asset.
+ */
+function updateVersion(gltf, options) {
+    options = defaultValue(options, {});
+    var targetVersion = options.targetVersion;
+    var version = gltf.version;
+    if (defined(gltf.asset)) {
+        version = defaultValue(version, gltf.asset.version);
+    }
+    if (!defined(version)) {
+        throw new DeveloperError('glTF must have a defined version');
+    }
+    var updateFunction = updateFunctions[version];
+    while (defined(updateFunction)) {
+        updateFunction(gltf);
+        version = gltf.asset.version;
+        if (version === targetVersion) {
+            break;
+        }
+        updateFunction = updateFunctions[version];
+    }
+    return gltf;
+}
+
+function glTF_0_8_to_glTF_1_0(gltf) {
```

Comment:
Do you have a suggested alternative? I was having a hard time making it still readable without underscores.